package.json

```diff
@@ -109,6 +111,7 @@
     "**/trim": "^1.0.0",
     "**/prismjs": "^1.24.0",
     "**/glob-parent": "^5.1.2",
-    "**/browserslist": " ^4.16.5"
+    "**/browserslist": " ^4.16.5",
+    "storybook-addon-i18next/react-i18next": "^11.8.8"
```

Comment:
Important! You can only have one react-i18next version installed, otherwise you end up with locales stored in different stores unable to access each other. I resolved this by using yarn's [selective resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) feature. See isaachinman/next-i18next#324 (comment)